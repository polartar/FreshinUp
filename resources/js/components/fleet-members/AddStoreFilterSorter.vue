<template>
  <div class="px-4 pt-4">
    <v-text-field
      v-model="name"
      label="Search by Fleet Member name"
      single-line
      outline
    />
    <div>
      <div class="pb-4">
        <button
          class="font-weight-bold grey--text"
          @click="toggleShowFilter"
        >
          <span v-if="showFilters">
            <v-flex>
              <v-icon small>fa-caret-down</v-icon>
              Hide Filters
            </v-flex>
          </span>
          <span v-else>
            <v-flex>
              <v-icon small>fa-caret-right</v-icon>
              Show Filters
            </v-flex>
          </span>
        </button>
      </div>
      <div
        v-show="showFilters"
      >
        <v-layout
          row
          wrap
          align-center
          justify-center
        >
          <v-flex
            sm3
            pr-2
          >
            <v-layout
              row
              justify-space-between
              mb-2
            >
              <filter-label
                class="grey--text"
              >
                State
              </filter-label>
              <clear-button
                v-if="state_of_incorporation"
                color="grey"
                @clear="state_of_incorporation = null"
              />
            </v-layout>
            <v-text-field
              v-model="state_of_incorporation"
              label="State of incorporation"
              outline
              single-line
            />
          </v-flex>
          <v-flex
            sm3
            pr-2
          >
            <v-layout
              row
              justify-space-between
              mb-2
            >
              <filter-label
                class="grey--text"
              >
                Type
              </filter-label>
              <clear-button
                v-if="type_id"
                color="grey"
                @clear="type_id = null"
              />
            </v-layout>
            <v-select
              v-model="type_id"
              :items="types"
              item-value="id"
              item-text="name"
              label="Type"
              outline
              single-line
            />
          </v-flex>
          <v-flex
            sm3
            pr-2
          >
            <v-layout
              row
              justify-space-between
              mb-2
            >
              <filter-label class="grey--text">
                Tags
              </filter-label>
              <clear-button
                v-if="tags && tags.length > 0"
                color="grey"
                @clear="tags = []; $refs.tag.resetTerm()"
              />
            </v-layout>
            <f-multi-simple
              ref="tag"
              v-model="tags"
              url="foodfleet/store-tags"
              term-param="filter[name]"
              results-id-key="uuid"
              placeholder="Search Tag"
              background-color="white"
              class="mt-0 pt-0 mb-4"
              height="48"
              not-clearable
              solo
              ouline
              flat
            />
          </v-flex>
          <v-flex
            sm3
          >
            <v-btn
              large
              depressed
              @click="clearFilters"
            >
              Clear all filters
            </v-btn>
          </v-flex>
        </v-layout>
      </div>
    </div>
  </div>
</template>

<script>
import ClearButton from '~/components/ClearButton'
import FilterLabel from '~/components/FilterLabel'
import FMultiSimple from 'fresh-bus/components/ui/FMultiSimple'
import MapValueKeysToData from '../../mixins/MapValueKeysToData'
export const DEFAULT_FILTERS = {
  name: '',
  state_of_incorporation: '',
  type_id: '',
  tags: []
}
export default {
  components: {
    ClearButton,
    FilterLabel,
    FMultiSimple
  },
  mixins: [MapValueKeysToData],
  props: {
    types: { type: Array, default: () => [] },
    // overriding value props from MapValueKeysToData mixin
    // to provide default value
    value: { type: Object, default: () => DEFAULT_FILTERS }
  },
  data () {
    return {
      showFilters: false,
      ...DEFAULT_FILTERS
    }
  },
  watch: {
    payload () {
      this.run()
    }
  },
  methods: {
    clearFilters () {
      this.name = ''
      this.state_of_incorporation = ''
      this.type_id = ''
      this.tags = []
      this.$refs.tag.resetTerm()
    },
    toggleShowFilter () {
      this.showFilters = !this.showFilters
    },
    run () {
      this.$emit('input', {
        name: this.payload.name,
        state_of_incorporation: this.payload.state_of_incorporation,
        type_id: this.payload.type_id,
        tag: this.tags ? this.tags.map(item => item.uuid) : null
      })
    }
  }
}
</script>
