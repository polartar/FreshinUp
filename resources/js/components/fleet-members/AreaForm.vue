<template>
  <form class="py-2">
    <v-flex
      xs12
      px-2
    >
      <div class="mb-2 text-uppercase grey--text font-weight-bold">
        Area address
      </div>
      <v-text-field
        v-model="name"
        placeholder="Enter an address"
        single-line
        outline
      />
    </v-flex>
    <v-flex
      xs12
      px-2
    >
      <div class="text-uppercase grey--text font-weight-bold">
        Radius
      </div>
      <v-slider
        v-model="radius"
        :min="0"
        :max="100"
        thumb-label
      />
    </v-flex>
    <v-flex
      xs12
      pa-2
    >
      <div class="text-xs-center">
        MAP HERE
      </div>
    </v-flex>
    <v-layout
      align-center
      justify-space-between
    >
      <v-btn
        depressed
        disabled
        @click="onCancel"
      >
        Cancel
      </v-btn>
      <v-btn
        depressed
        color="primary"
        @click="save"
      >
        Save changes
      </v-btn>
    </v-layout>
  </form>
</template>
<script>
import MapValueKeysToData from '../../mixins/MapValueKeysToData'
export default {
  mixins: [MapValueKeysToData],
  data () {
    return {
      id: 0,
      name: '',
      radius: 0,
      state: '',
      store_uuid: ''
    }
  },
  methods: {
    onCancel () {
      this.$emit('cancel')
    }
  }
}
</script>
