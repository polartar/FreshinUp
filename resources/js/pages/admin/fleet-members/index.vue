<template>
  <div>
    <v-flex
      d-flex
      align-center
      justify-space-between
      ma-2
    >
      <h2 class="white--text">
        {{ pageTitle }}
      </h2>
      <v-flex text-xs-right>
        <v-btn
          slot="activator"
          color="primary"
          dark
          @click="storeNew"
        >
          Add New Fleet Member
        </v-btn>
      </v-flex>
    </v-flex>

    <search-filter-sorter
      without-filter-label
      without-sort-by
      @run="runFilter"
    />

    <store-list
      v-if="!isLoading"
      :stores="stores"
      :statuses="statuses"
      :is-loading="isLoading || isLoadingList"
      :rows-per-page="pagination.rowsPerPage"
      :page="pagination.page"
      :total-items="pagination.totalItems"
      :sort-by="sorting.sortBy"
      :descending="sorting.descending"
      @paginate="onPaginate"
      @change-status="changeStatus"
      @manage-view="storeView"
      @manage-edit="storeEdit"
      @manage-delete="deleteStore"
      @manage-multiple-delete="deleteStore"
      @change_status_multiple="changeStatusMultiple"
    />
    <v-dialog
      v-model="deleteDialog"
      max-width="500"
    >
      <simple-confirm
        :class="{ 'deleting': deletablesProcessing }"
        :title="deleteDialogTitle"
        ok-label="Yes"
        cancel-label="No"
        @ok="onSubmitDelete"
        @cancel="onCancelDelete"
      >
        <div class="py-5 px-2">
          <template v-if="deletablesProcessing">
            <div class="text-xs-center">
              <p class="subheading">
                Processing, please wait...
              </p>
              <v-progress-circular
                :rotate="-90"
                :size="200"
                :width="15"
                :value="deletablesProgress"
                color="primary"
              >
                {{ deletablesStatus }}
              </v-progress-circular>
            </div>
          </template>
          <template v-else>
            <p class="subheading">
              <span v-if="deletables.length < 2">Store</span>
              <span v-else> Stores</span>
              : {{ deleteTemp | formatDeleteTitles }}
            </p>
          </template>
        </div>
      </simple-confirm>
    </v-dialog>
  </div>
</template>

<script>
import { mapGetters, mapActions, mapState } from 'vuex'
import { deletables } from 'fresh-bus/components/mixins/Deletables'
import StoreList from '~/components/stores/StoreList.vue'
import simpleConfirm from 'fresh-bus/components/SimpleConfirm.vue'
import get from 'lodash/get'
import SearchFilterSorter from 'fresh-bus/components/search/filter-sorter.vue'

export default {
  layout: 'admin',
  components: {
    StoreList,
    simpleConfirm,
    SearchFilterSorter
  },
  filters: {
    formatDeleteTitles (value) {
      return value.map(item => item.name).join(', ')
    }
  },
  mixins: [deletables],
  data () {
    return {
      pageTitle: 'Fleet Member List',
      deleteDialog: false,
      lastFilterParams: {
        sort: '-created_at'
      },
      deleteTemp: []
    }
  },
  computed: {
    isLoadingList () {
      return get(this.$store, 'state.stores.pending.items', true)
    },
    ...mapGetters('stores', {
      stores: 'items',
      pagination: 'pagination',
      sorting: 'sorting',
      sortBy: 'sortBy'
    }),
    ...mapGetters('page', ['isLoading']),
    ...mapState('stores', ['sortables']),
    ...mapGetters('storeStatuses', { 'statuses': 'items' }),
    deleteDialogTitle () {
      return this.deleteTemp.length < 2 ? 'Are you sure you want to delete this store?' : 'Are you sure you want to delete the following stores?'
    }
  },
  methods: {
    ...mapActions('page', {
      setPageLoading: 'setLoading'
    }),
    storeNew () {
      alert('coming soon')
    },
    storeView (store) {
      alert('coming soon')
    },
    storeEdit (store) {
      alert('coming soon')
    },
    deleteStore (stores) {
      if (!Array.isArray(stores)) {
        stores = [stores]
      }
      this.deleteTemp = stores
      this.deleteDialog = true
    },
    async onSubmitDelete () {
      this.deletablesProcessing = true
      this.deletablesProgress = 0
      this.deletablesStatus = ''
      let dispatcheables = []

      this.deleteTemp.forEach((store) => {
        dispatcheables.push(this.$store.dispatch('stores/deleteItem', { getItems: false, params: { id: store.uuid } }))
      })

      let chunks = this.chunk(dispatcheables, this.deleteTempParrallelRequest)
      let doneCount = 0

      for (let i in chunks) {
        await Promise.all(chunks[i])
        doneCount += chunks[i].length
        this.deleteTempStatus = doneCount + ' / ' + this.deleteTemp.length + ' Done'
        this.deleteTempProgress = doneCount / this.deleteTemp.length * 100
        await this.sleep(this.deletablesSleepTime)
      }

      this.filterStores(this.lastFilterParams)
      await this.sleep(500)
      this.deletablesProcessing = false
      this.deleteDialog = false
    },
    onCancelDelete () {
      this.deleteDialog = false
      this.deleteTemp = []
    },
    changeStatus (status, store) {
      this.$store.dispatch('stores/patchItem', { data: { status }, params: { id: store.uuid } }).then(() => {
        this.filterStores(this.lastFilterParams)
      })
    },
    changeStatusMultiple (status, stores) {
      stores.forEach((store) => {
        this.changeStatus(status, store)
      })
    },
    onPaginate (value) {
      this.$store.dispatch('stores/setPagination', value)
      this.$store.dispatch('stores/getItems')
    },
    runFilter(params) {
      this.filterStores({
        name: params.term,
        ...this.lastFilterParams
      })
    },
    filterStores (params) {
      this.lastFilterParams = params
      this.$store.dispatch('stores/setSort', params.sort)
      this.$store.dispatch('stores/setFilters', {
        ...this.$route.query,
        ...this.lastFilterParams
      })
      this.$store.dispatch('stores/getItems')
    }
  },
  beforeRouteEnterOrUpdate (vm, to, from, next) {
    vm.setPageLoading(true)

    Promise.all([
      vm.$store.dispatch('stores/setFilters', {
        ...vm.$route.query
      }),
      vm.$store.dispatch('storeStatuses/getItems')
    ])
      .then(() => {
        vm.$store.dispatch('page/setLoading', false)
        if (next) next()
      })
  }
}
</script>
