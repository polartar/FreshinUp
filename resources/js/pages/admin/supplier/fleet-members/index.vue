<script>
  import Page, { INCLUDE } from '~/pages/admin/fleet-members/index.vue'
  import { mapGetters } from 'vuex'
  export default {
    extends: Page,
    computed: {
      ...mapGetters(['currentUser']),
      ...mapGetters('suppliers/stores', {
        stores: 'items',
        pagination: 'pagination',
        sorting: 'sorting',
        sortBy: 'sortBy',
        storesLoading: 'itemsLoading'
      }),
      newLink () {
        return '/admin/supplier/fleet-members/new'
      },
    },
    async beforeRouteEnterOrUpdate (vm, to, from, next) {
      vm.setPageLoading(true)
      await vm.$store.dispatch('currentUser/getCurrentUser')
      Promise.all([
        vm.$store.dispatch('suppliers/stores/setFilters', {
          ...vm.$route.query,
          ...this.lastFilterParams
        }),
        vm.$store.dispatch('suppliers/stores/getItems', {
          params: {
            supplierId: vm.currentUser.uuid
          }
        }),
        vm.$store.dispatch('storeStatuses/getItems')
      ])
        .then()
        .catch((error) => console.error(error))
        .then(() => {
          vm.setPageLoading(false)
          if (next) next()
        })
    },
    methods: {
      onPaginate (value) {
        this.$store.dispatch('suppliers/stores/setPagination', value)
        this.$store.dispatch('suppliers/stores/getItems', {
          params: {
            supplierId: this.currentUser.uuid
          }
        })
      },
      storeViewOrEdit (store) {
        this.$router.push({ path: `/admin/supplier/fleet-members/${store.uuid}/edit` })
      },
      filterStores (params) {
        this.lastFilterParams = params
        this.$store.dispatch('suppliers/stores/setFilters', {
          ...this.$route.query,
          ...this.lastFilterParams
        })
        this.$store.dispatch('suppliers/stores/getItems', {
          params: {
            supplierId: this.currentUser.uuid,
            include: INCLUDE
          }
        })
      }
    }
  }
</script>
