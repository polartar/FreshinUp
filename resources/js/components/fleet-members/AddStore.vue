<template>
  <div>
    <div class="px-2 py-2">
      <filter-sorter
        :types="types"
        @input="runFilters"
      />
    </div>
    <div
      class="px-2"
      style="border-top: 1px solid gainsboro;"
    >
      <f-data-table
        :headers="headers"
        :items="stores"
        :multi-item-actions="multipleItemActions"
        item-key="uuid"
        hide-actions
        select-all
        v-bind="$attrs"
        v-on="$listeners"
      >
        <template v-slot:item-inner-name="{ item }">
          <div style="position: relative;">
            <v-tooltip top>
              <template v-slot:activator="{ on, attrs }">
                <a
                  class="primary--text"
                  href="#"
                  @click.prevent="viewItem"
                >
                  {{ get(item, 'name') }}
                </a>
              </template>
              View fleet member details (new tab)
            </v-tooltip>
          </div>
        </template>
        <template v-slot:item-inner-tags="{ item }">
          <div>
            <v-chip
              v-for="(tag, index) of get(item, 'tags', [])"
              :key="index"
              color="secondary"
              text-color="white"
            >
              {{ tag.name }}
            </v-chip>
          </div>
        </template>
        <template v-slot:item-inner-state_of_incorporation="{ item }">
          <div class="grey--text">
            {{ get(item, 'state_of_incorporation') }}
          </div>
        </template>
        <template v-slot:item-inner-manage="{ item }">
          <v-btn
            :depressed="manageButtonLabel(item) !== 'Assign'"
            :disabled="!isEligible(item)"
            :class="manageButtonClass(item)"
            @click="onManageClicked(item)"
          >
            {{ manageButtonLabel(item) }}
          </v-btn>
        </template>
      </f-data-table>
    </div>
  </div>
</template>
<script>
import get from 'lodash/get'
import FDataTable from '@freshinup/core-ui/src/components/FDataTable'
import AddStoreFilterSorter from './AddStoreFilterSorter'

export const HEADERS = [
  { text: 'Fleet member', value: 'name' },
  { text: 'State of incorporation', value: 'state_of_incorporation' },
  { text: 'Tags', value: 'tags' },
  { text: 'Manage', value: 'manage' }
]

export const MULTIPLE_ITEM_ACTIONS = []
export default {
  components: {
    FDataTable,
    FilterSorter: AddStoreFilterSorter
  },
  props: {
    headers: { type: Array, default: () => HEADERS },
    multipleItemActions: { type: Array, default: () => MULTIPLE_ITEM_ACTIONS },
    event: { type: Object, required: true, default: () => ({}) },
    stores: { type: Array, default: () => [] },
    types: { type: Array, default: () => [] }
  },
  data () {
    return {
      showFilters: false
    }
  },
  methods: {
    get,
    runFilters (payload) {
      this.$emit('run-filter', payload)
    },
    viewItem (item) {
      this.$emit('manage', 'view', item)
      this.$emit('manage-view', item)
    },
    // TODO: methods need more attention at some point later in the future
    hasBookedAnEvent (member) {
      const stores = get(member, 'event_stores', [])
      return stores.findIndex(e => e.event_uuid !== this.event.uuid) !== -1
    },
    isEligible (member) {
      return !this.hasBookedAnEvent(member) && !member['has_expired_licences_docs']
    },
    isAssignedToThisEvent (member) {
      const stores = get(member, 'event_stores', [])
      return stores.findIndex(e => e.event_uuid === this.event.uuid) !== -1
    },
    isDeclined (member) {
      const stores = get(member, 'event_stores', [])
      return stores.findIndex(e => e.event_uuid === this.event.uuid && e.declined) !== -1
    },
    manageButtonLabel (item) {
      if (this.isEligible(item)) {
        if (this.isDeclined(item)) { return 'Declined' }
        if (!this.isAssignedToThisEvent(item)) { return 'Assign' } else { return 'Assigned' }
      }
      if (this.hasBookedAnEvent(item)) { return 'Booked' }
      if (item['has_expired_licences_docs']) { return 'Expired' }
      return ''
    },
    onManageClicked (item) {
      if (this.manageButtonLabel(item) === 'Assign') {
        this.$emit('manage', 'assign', item)
        this.$emit('manage-assign', item)
      }
    },
    manageButtonClass (item) {
      const label = this.manageButtonLabel(item)
      return {
        'primary': label === 'Assign',
        'blue-grey lighten-5': ['Expired', 'Booked'].includes(label),
        'blue-grey lighten-3 white--text': ['Declined', 'Assigned'].includes(label)
      }
    }
  }
}
</script>
