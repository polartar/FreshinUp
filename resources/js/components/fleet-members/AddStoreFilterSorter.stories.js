import AddStoreFilterSorter from './AddStoreFilterSorter'
import { storiesOf } from '@storybook/vue'
import { FIXTURE_STORE_TYPES } from '../../../../tests/Javascript/__data__/storeTypes'
import { action } from '@storybook/addon-actions'
import { FIXTURE_STORE_TAGS } from '../../../../tests/Javascript/__data__/storeTags'

export const FIXTURE_ADD_STORE_FILTERS = {
  name: 'California',
  state_of_incorporation: 'Magnolia',
  type_id: 1,
  tags: FIXTURE_STORE_TAGS
}

export const Default = () => ({
  components: {
    FilterSorter: AddStoreFilterSorter
  },
  template: `
    <filter-sorter/>
  `
})

export const Populated = () => ({
  components: {
    FilterSorter: AddStoreFilterSorter
  },
  data () {
    return {
      value: FIXTURE_ADD_STORE_FILTERS,
      types: FIXTURE_STORE_TYPES
    }
  },
  methods: {
    runFilters (payload) {
      action('run-filter')(payload)
    },
  },
  template: `
    <filter-sorter
      :value="value"
      :types="types"
      @run-filter="runFilters"
    />
  `
})

storiesOf('FoodFleet|components/fleet-members/AddStoreFilterSorter', module)
  .add('Default', Default)
  .add('Populated', Populated)
