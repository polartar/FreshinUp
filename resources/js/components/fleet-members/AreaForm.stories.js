import { storiesOf } from '@storybook/vue'
import { action } from '@storybook/addon-actions'

import AreaForm from './AreaForm'
import { FIXTURE_STORE } from '../../../../tests/Javascript/__data__/stores'

export const Default = () => ({
  components: { AreaForm },
  methods: {
    onSave () {
      action('save')()
    },
    onCancel () {
      action('cancel')()
    }
  },
  template: `
    <v-container>
      <area-form
        @input="onSave"
        @cancel="onCancel"/>
    </v-container>
  `
})

export const WithData = () => ({
  components: { AreaForm },
  data () {
    return {
      store: FIXTURE_STORE
    }
  },
  methods: {
    onSave () {
      action('save')()
    },
    onCancel () {
      action('cancel')()
    }
  },
  template: `
    <v-container>
      <area-form
        :value="store"
        @input="onSave"
        @cancel="onCancel"/>
    </v-container>
  `
})

storiesOf('FoodFleet|components/fleet-members/AreaForm', module)
  .addParameters({
    backgrounds: [
      { name: 'default', value: '#f1f3f6', default: true }
    ]
  })
  .add('Default', Default)
  .add('WithData', WithData)
