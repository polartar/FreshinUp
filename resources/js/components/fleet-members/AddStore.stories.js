import { storiesOf } from '@storybook/vue'

import AddStore from './AddStore'

import { FIXTURE_STORES } from '../../../../tests/Javascript/__data__/stores'
import { FIXTURE_EVENT } from '../../../../tests/Javascript/__data__/event'
import { FIXTURE_STORE_TYPES } from '../../../../tests/Javascript/__data__/storeTypes'

export const Default = () => ({
  components: { AddStore },
  data () {
    return {
      event: FIXTURE_EVENT
    }
  },
  template: `
    <v-container>
      <add-store :event="event"/>
    </v-container>
  `
})

export const WithData = () => ({
  components: { AddStore },
  data () {
    return {
      stores: FIXTURE_STORES,
      types: FIXTURE_STORE_TYPES,
      event: FIXTURE_EVENT
    }
  },
  template: `
    <v-container>
      <add-store
        :stores="stores"
        :store-types="types"
        :event="event"/>
    </v-container>
  `
})

storiesOf('FoodFleet|components/event/AddStore', module)
  .add('Default', Default)
  .add('WithData', WithData)
