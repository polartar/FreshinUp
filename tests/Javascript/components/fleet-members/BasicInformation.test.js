import { mount } from '@vue/test-utils'
import createLocalVue from 'vue-cli-plugin-freshinup-ui/utils/testing/createLocalVue'
import Component from '~/components/fleet-members/BasicInformation.vue'
import { FIXTURE_FLEET_MEMBER } from 'tests/__data__/fleet-members'

describe('flee-members/BasicInformation', () => {
  let localVue

  describe('Snapshots', () => {
    beforeEach(() => {
      const vue = createLocalVue({ validation: true })
      localVue = vue.localVue
    })
    test('default', () => {
      const wrapper = mount(Component, {
        localVue: localVue
      })
      expect(wrapper.element).toMatchSnapshot()
    })
    test('member set', () => {
      const wrapper = mount(Component, {
        localVue: localVue,
        propsData: {
          member: FIXTURE_FLEET_MEMBER
        }
      })
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
