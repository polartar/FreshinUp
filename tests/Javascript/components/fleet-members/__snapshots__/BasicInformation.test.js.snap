// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`flee-members/BasicInformation Snapshots default 1`] = `
<div>
  Fleet member basic information component
</div>
`;

exports[`flee-members/BasicInformation Snapshots member set 1`] = `
<div>
  Fleet member basic information component
</div>
`;
